lib/Cron/Daemon.php

```diff
@@ -83,9 +83,7 @@ private function getWorkers() {
     yield $this->workersFactory->createSubscribersEngagementScoreWorker();
     yield $this->workersFactory->createSubscribersLastEngagementWorker();
     yield $this->workersFactory->createSubscribersCountCacheRecalculationWorker();
-    if ($this->featureSwitch->isSupported(FeaturesController::RE_ENGAGEMENT_EMAIL)) {
```

Comment:
I think that this was the only usage of `$this->featureSwitch` within the class. Could you please also remove the dependency from the constructor?